pages/[[server]]/@[account]/index/index.vue

```diff
@@ -21,11 +21,12 @@ if (account) {
     title: () => `${t('account.posts')} | ${getDisplayName(account)} (@${account.acct})`,
   })
 }
+const stream = useStreaming(client => client.user.subscribe())
 </script>
 
 <template>
   <div>
     <AccountTabs />
-    <TimelinePaginator :paginator="paginator" :preprocess="reorderAndFilter" context="account" :account="account" />
+    <TimelinePaginator :paginator="paginator" :preprocess="reorderAndFilter" context="account" :account="account" :stream="stream" />
```

Comment:
Re-checking the PR. Is this change needed? Isn't the user stream unrelated to the `[[server]]/@[account]` here?

Reply:
Sorry, I will check the API

Reply:
I have checked this, I don't think user stream is unrelated to [[server]]/@[account], when the user delete a status in his/her account, we should delete the same status in user stream(which will) be shown in home timeline page.Maybe it looks wired...

Reply:
The perfact way is that we can use access the same stream instance in the home timeline, but I have no idea to get access to home steam...

Reply:
Maybe we can use pinia or attach the stream into Vue prototype,